### ISSUE TITLE
Better error messages when misconfiguring multiple nics

### ISSUE THREAD

**Issue by liorkesos** [harvester/harvester#531]
I installed harvester on a machine with 4 network cards.
Apparently once I disconnected the cable I reconnected it to the wrongn network connection (eth3 insteadf of eth0)
When installing you guys showed which nic was connected and that was helpfull.
Maybe you should have similar information in your status page.
"ETH0 confugured but ETH3 connected."

This is the error I sae in the the k3s log
```
 Determining IP address of specified interface: eth0
time="2021-03-14T15:49:11.821154098Z" level=fatal msg="flannel exited: failed to find IPv4 address for interface eth0"
```

**Comment by guangbochen** [comment]
considering to add the `Network` tab on the host view page and print all valid information of the available NICs, e.g name, status, mac address and etc.
![image](https://user-images.githubusercontent.com/4569037/138252453-e79bbea7-e905-4c70-9aa3-3c60fedae1c2.png)

**Comment by n313893254** [comment]
Verify Steps:
- Install the latest harvester
- Go to the host detail page
- Check Network tabs

Addtional Context:
![image.png](https://images.zenhubusercontent.com/59db4297b0222d5de47ad632/3b3463ec-a755-4589-b00c-ac8c407bc55e)

**Comment by TachunLin** [comment]
Verified **fixed** on master-119c2c48-head. Close this issue.

### Result
On the Host view page, now we can see detailed network status including `Name`, `Type`, `IP Address`, `Status` etc..
![image](https://user-images.githubusercontent.com/29251855/141070311-55ec4382-d777-4289-91c7-cebe81db3356.png)

### Test Information
* Test Environment: 3 nodes harvester on provo bare machines (RackD8 ClusterA)
* Harvester version: master-119c2c48-head

### Verify Steps

1. Enable vlan cluster network setting and set a default network interface
2. Wait a while for the setting take effect on all harvester nodes
3. Click nodes on host page 
4. Check the network tab

### Additional Context
None
